articles/libraries/lock/v11/migration-guide.md

```diff
@@ -41,7 +41,7 @@ If you have any questions or concerns, you can discuss them in the [Auth0 Commun
 
 ## Disabling legacy Lock API
 
-After you update to Lock v11 and/or Auth0.js v9, it is advised that you turn off the **Legacy Lock API** toggle in the Dashboard. This will make your Auth0 tenant behave as if the legacy API is no longer available. Starting on July 16, 2018, this option will be forcibly disabled, so it is recommended you opt-in before that time to verify that your configuration will work correctly. 
```

Comment:
I wouldn't change this message as part of this PR. Although Hilary provided the text for the Legacy Lock check, I'm not sure she wants to change this text. In my opinion there's clear information here about the toggle being forcibly disabled by July 16th, I wouldn't remove it.

Reply:
The text was so similar that we thought it was replacement text. I see your point, though, and edited to make sure all necessary info was included.